.github/workflows/main.yml

```diff
@@ -2,7 +2,9 @@ name: Build
 
 on:
   push:
-    branches: [ main ]
+    branches:
```

Comment:
in pdns, this just works, I believe:

```
on:
  push:
  pull_request:
  schedule:
    - cron: '0 22 * * 3'
```

(the cron is unrelated but is probably a good idea)